Dockerfile

```diff
@@ -1,11 +1,23 @@
 FROM ubuntu:22.04
```

Comment:
another couple of suggestions:
 - ECR is updated more frequently than the official Ubuntu image in Docker Hub, so if you want to use that one instead, it's `public.ecr.aws/ubuntu/ubuntu:22.04`
 - if you're planning on using `renovate` in this repo, then you might want to ping a digest, i.e. `ubuntu:22.04@${digest}`